packages/relic/lib/src/headers/vary_header.dart

```diff
@@ -0,0 +1,65 @@
+part of '../headers.dart';
+
+/// A class representing the HTTP Vary header.
+///
+/// This class manages the list of headers that the response may vary on.
+/// It provides functionality to parse, manipulate, and generate Vary header values.
+class VaryHeader {
+  /// A list of headers that the response varies on.
+  final List<String> fields;
```

Comment:
The name in the specification is `header-name`, not `field`. Also, this is just a `List<String>`, I think we should use that in our header, rather than wrapping it in a custom class. If we are keeping the class. We should add a second constructor `VaryHeader.uncachable()` that returns `['*']`.